java/ql/src/change-notes/2021-01-02-netty-response-splitting-improve.md

```diff
@@ -0,0 +1,9 @@
+---
+category: minorAnalysis
```

Comment:
Renaming the `@id` is probably fine, but I think the change note needs to be more explicit about this, at the very least also mentioning the previous `@id`. Also, since from that perspective this is essentially a deletion plus introduction of a query, the category should be `breaking`, I think, as that's the relevant category for a query deletion.

Reply:
Resolved! Thanks!